game/requesthandler.cc

```diff
@@ -95,6 +95,10 @@ void RequestHandler::Get(web::http::http_request request)
             m_songs.sortSpecificChange(3);
         } else if(query == "sort=edition&order=descending") {
             m_songs.sortSpecificChange(3, true);
+        } else if(query == "sort=creator&order=ascending") {
+            m_songs.sortSpecificChange(10);
+        } else if(query == "sort=creator&order=descending") {
+            m_songs.sortSpecificChange(10, true);
```

Comment:
Hmmm better would be to change `sortSpecificChange` to accept an enum. This way we don't spread magic numbers throughout the code.
If it's too much for this, then please create an issue mentioning this refactor so we can do it later.

The changes itself work :)

Reply:
It might be better to do that in a separate PR indeed, as also the `addSongOrder(..)` and other related code needs to work with these enum class values.

Reply:
Created issue: #904